sdk/src/main/java/co/omise/android/ui/InstallmentTermChooserFragment.kt

```diff
@@ -28,20 +33,58 @@ internal class InstallmentTermChooserFragment : OmiseListFragment<InstallmentTer
     }
 
     override fun listItems(): List<InstallmentTermResource> {
+        val currency = requester!!.currency
+        val minimumInstallmentAmountPerType =
+            mapOf(
+                SourceType.Installment.Bay to Amount.fromLocalAmount(500.0, currency),
+                SourceType.Installment.FirstChoice to Amount.fromLocalAmount(300.0, currency),
+                SourceType.Installment.Bbl to Amount.fromLocalAmount(500.0, currency),
+                SourceType.Installment.Mbb to Amount.fromLocalAmount(83.33, currency),
+                SourceType.Installment.Ktc to Amount.fromLocalAmount(300.0, currency),
+                SourceType.Installment.KBank to Amount.fromLocalAmount(300.0, currency),
+                SourceType.Installment.Scb to Amount.fromLocalAmount(500.0, currency),
+                SourceType.Installment.Ttb to Amount.fromLocalAmount(500.0, currency),
+                SourceType.Installment.Uob to Amount.fromLocalAmount(500.0, currency),
+            )
+        val interestRatePerType =
+            mapOf(
+                SourceType.Installment.Bay to 0.0074,
+                SourceType.Installment.FirstChoice to 0.0116,
+                SourceType.Installment.Bbl to 0.0074,
+                SourceType.Installment.Mbb to 0.0,
+                SourceType.Installment.Ktc to 0.0074,
+                SourceType.Installment.KBank to 0.0065,
+                SourceType.Installment.Scb to 0.0074,
+                SourceType.Installment.Ttb to 0.008,
+                SourceType.Installment.Uob to 0.0064,
+            )
         return installment
             ?.installmentTerms
             .orEmpty()
-            .map {
+            .filter { term ->
+                val sourceType = (installment?.backendType as? BackendType.Source)?.sourceType
+                val minimumAmount = minimumInstallmentAmountPerType[sourceType]
+                val req = requester
+                val amount = req!!.amount
+                var interestAmount = 0.0
+                if (zeroInterestInstallments == false) {
+                    val rate = interestRatePerType[sourceType] ?: 0.0
+                    interestAmount = amount.toDouble() * rate
+                }
+                val installmentAmountPerMonth = (amount / term) + interestAmount
```

Comment:
Since interest rate is calculated from the total, it needs to be added to the amount before dividing by terms.

```suggestion
                val installmentAmountPerMonth = (amount + interestAmount) / term
```